OpenBCI_32bit_Library.cpp

```diff
@@ -914,6 +914,7 @@ void OpenBCI_32bit_Library::processIncomingSampleRate(char c) {
 void OpenBCI_32bit_Library::processInsertMarker(char c) {
   markerValue = c;
   newMarkerReceived = true;
+  activateLedMarker(true);
```

Comment:
should this be `ON` to match line 625?

Reply:
Your call - let me know which seems clearer to you.
activateLedMarker is a sequence that will run through switching the led off then on.
I have used true and false to signal whether a sequence is active.  
The line is 625 is a LED command (on or off) and for that I used ON/OFF.